### PR TITLE
invoke* use base64 script

### PR DESCRIPTION
**Pull request by Tommo-L**
Fix https://github.com/neo-project/neo-modules/issues/337

### CODE REVIEW

File: tests/Neo.Network.RPC.Tests/UT_RpcClient.cs [hunk 1]
```diff
@@ -286,9 +286,11 @@ public void TestInvokeFunction()
         [TestMethod]
         public void TestInvokeScript()
         {
-            var test = TestUtils.RpcTestCases.Find(p => p.Name == nameof(rpc.InvokeScript).ToLower());
-            var result = rpc.InvokeScript(test.Request.Params[0].AsString().HexToBytes());
-            Assert.AreEqual(test.Response.Result.ToString(), result.ToJson().ToString());
+            //var test = TestUtils.RpcTestCases.Find(p => p.Name == nameof(rpc.InvokeScript).ToLower());
```
Comment (shargon) [inline tests/Neo.Network.RPC.Tests/UT_RpcClient.cs:289]: Remove them

Comment (Tommo-L) [reply tests/Neo.Network.RPC.Tests/UT_RpcClient.cs]: Done

**Review by shargon — APPROVED** [review]

**Comment by devhawk** [comment]
Can this please wait until after #335 is merged?

File: src/RpcServer/RpcServer.Wallet.cs [hunk 2]
```diff
@@ -341,11 +341,6 @@ private JObject SignAndRelay(Transaction tx)
 
         internal static UInt160 AddressToScriptHash(string address)
         {
-            if (UInt160.TryParse(address, out var scriptHash))
```
Comment (cloud8little) [inline src/RpcServer/RpcServer.Wallet.cs:344]: this change should remove now.

Comment (Tommo-L) [reply src/RpcServer/RpcServer.Wallet.cs]: Done

**Comment by cloud8little** [comment]
when processing with wallet open, this code should also be updated. It can not use script anymore, but decode from base64
https://github.com/neo-project/neo-modules/blob/81e07af27db214cca0f7287a81041ca79e201379/src/RpcServer/RpcServer.Wallet.cs#L185

**Comment by cloud8little** [comment]
BTW, for consistency, shall we update neo-cli too? @Tommo-L @shargon

**Comment by Tommo-L** [comment]
> @Tommo-L when processing with wallet open, this code should also be updated. It can not use script anymore, but decode from base64
> https://github.com/neo-project/neo-modules/blob/81e07af27db214cca0f7287a81041ca79e201379/src/RpcServer/RpcServer.Wallet.cs#L185

Done

**Comment by chenzhitong** [comment]
> BTW, for consistency, shall we update neo-cli too? @Tommo-L @shargon

pull request: https://github.com/neo-project/neo-node/pull/673

**Comment by ProDog** [comment]
And here?
https://github.com/neo-project/neo/blob/6886991ca67f2c2148429ba87bf7d782f8cb12c8/src/neo/Wallets/Wallet.cs#L353

**Comment by Tommo-L** [comment]
> And here?
> https://github.com/neo-project/neo/blob/6886991ca67f2c2148429ba87bf7d782f8cb12c8/src/neo/Wallets/Wallet.cs#L353

Fixed https://github.com/neo-project/neo/pull/2005

**Review by shargon — DISMISSED** [review]

**Comment by cloud8little** [comment]
Test passed. 
1. RpcClient: InvokeFunctionAsync with/without signer and InvokeScriptAsync with/without signer.
2. RpcServer: InvokeScript and Invokefunction both works with/without opening wallet.

**Review by shargon — APPROVED** [review]

**Pull request merged**